packages/components/spinner/src/spinner.css

```diff
@@ -0,0 +1,107 @@
+/* General........................................................ */
+
+:host {
+	--border-large: 10px;
+	--border-medium: 6px;
+	--border-small: 2px;
+	--size-large: 100px;
+	--size-medium: 50px;
+	--size-small: 20px;
+	--border-color-light: var(--ts-color-gray-lighter);
+	--border-color-dark: var(--ts-color-gray-lighter);
+	--spinner-blue-light: var(--ts-color-blue);
+	--spinner-blue-dark: var(--ts-color-blue);
+	--spinner-mono-light: var(--ts-color-slate-lightest);
+	--spinner-mono-dark: var(--ts-color-white);
+	z-index: var(--ts-zindex-spin);
+}
+
+:host .spinner {
+	border: var(--border-large) solid #f3f3f3; /* Light grey */
+	border-top: var(--border-large) solid #3498db; /* Blue */
+	border-radius: 50%;
+	width: var(--size-large);
+	height: var(--size-large);
+	animation: spin 1s linear infinite;
+	margin: 0 auto;
+}
+
+:host .message {
+	text-align: center;
+	margin: var(--ts-unit);
+	font-size: var(--ts-font-size-default);
+}
+/* Size............................................................ */
+
+:host([data-size='large']) .spinner {
```

Comment:
When you are using selectors like this `:host([some-property])` you need to set the reflect to true in the props defining so, when someone use js to create the element also works

Reply:
https://github.com/Tradeshift/elements/wiki/Common-gotchas#add-reflect-true-for-all-properties